src/metrocs/redistricting/StarGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package metrocs.redistricting;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+/**
+ * Tests for Voter objects.
+ *
+ * @author Jada, David, Colin, John, Daniel, Ali
+ * @version
+ */
+public class StarGeneratorTest {
+    @Test
+    public void generatePatternTest() {
```

Comment:
This method name indicates that this is a more complete test of the method. Perhaps it should be renamed to be more specific as to what is being tested/affirmed.